### PR TITLE
feat: table header themes update

### PR DESCRIPTION
**Pull request by michalrozenek**
Table component
TableHeader themes additions

- primaryLight
- white

Ticket:  https://atomlearningltd.atlassian.net/browse/NC-442

Screenshots: 

Zrzut ekranu 2024-02-5 o 09 20 04
Zrzut ekranu 2024-02-5 o 09 20 26

Types improvement :)

**Before:**

- no support for themes type

Zrzut ekranu 2024-02-5 o 15 11 32

**After:**

Zrzut ekranu 2024-02-5 o 15 13 35
Zrzut ekranu 2024-02-5 o 15 13 56

Zrzut ekranu 2024-02-5 o 15 47 44

### CODE REVIEW

**Review by MiguelGDLR — APPROVED** [review]

File: lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap [hunk 1]
```diff
@@ -450,14 +450,14 @@ exports[`DateInput component renders lg size 1`] = `
     display: table;
   }
 
-  .c-dOwGHN-hztIub-isToday-true {
```
Comment (andoulla) [inline lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap:453]: why is this changing here?

Comment (michalrozenek) [reply lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap:453]: Very good question. I did nothing here. Just run `yarn test -u` and it happened.

Comment (andoulla) [reply lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap:453]: can you please check out main and see what happens if you run yarn test -u there?

I think we should look into it before merging

Comment (michalrozenek) [reply lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap:453]: Ok, so basically it just moves position of this declaration.
Not sure why. I checkout to `main` and it's same.
Still updating this line

Zrzut ekranu 2024-02-6 o 10 36 03

Comment (andoulla) [reply lib/src/components/date-input/__snapshots__/DateInput.test.tsx.snap:453]: so probably, someone didn't run all the snapshots when they were making a change in that component. it is weird that the CI didn't catch it

Thanks for looking into it

**Review by andoulla — APPROVED** [review]

**Pull request merged**